packages/dashboard/src/components/Card/animations/Shimmer.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+
+import { classNamesFunction } from 'office-ui-fabric-react/lib/Utilities';
+
+import { getStyles, IShimmerStyles } from './Shimmer.style';
+
+export interface IShimmerProps {}
+
+export class Shimmer extends React.Component<IShimmerProps> {
```

Comment:
Could the existing Shimmer component from office-ui-fabric-react be used instead?

Reply:
This component for animations only

Reply:
Could you clarify? What exactly is the difference between this new Shimmer component and the existing Fabric Shimmer component?